docs/tutorial.md

```diff
@@ -291,7 +293,7 @@ spec:
           name: skaffold-image-leeroy-web
   results:
     type: gcs
-    url: gcs://somebucket/results/logs
+    url: gcs://somebucket/results/logs #configure: remove results entirely if you're happy to use stdout
```

Comment:
note to whoever implements #386 that we might want to remove the `results` config from our docs since it currently does nothing :thinking: